Tests/TestHarness.psm1

```diff
@@ -0,0 +1,54 @@
+function Invoke-TestHarness
```

Comment:
It does have some behavior differences required because the resources and examples are all moved into a subfolder within the repo rather than being in the root. I've just copied the harness code almost straight over from SharePointDsc and xNetworking.